scripts/utils.py

```diff
@@ -0,0 +1,19 @@
+import re
```

Comment:
I would recommend removing this file completely and use the `packaging` libraries convenience utilities for normalization of names. I believe we are already using packaging in the other scripts. https://packaging.pypa.io/en/stable/utils.html

Reply:
Done! Thanks for that tip!